src/pages/getting-started/livechat-apps/index.mdx

```diff
@@ -46,6 +42,143 @@ See a 4-minute tutorial explaining how to create a simple app in the Developer C
+6. Click the **Continue button** and enjoy your new widget.
+
+<img
+  alt="Creating LiveChat apps step2.1"
+  src="/images/getting-started/livechat-apps/LiveChat -apps-step2.1.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 3 Display details**
+
+Now, it is time for the **Display details** tab that allows you to configure your app's short description and add an icon for better visibility in the Agent App.
+
+Follow the steps below to add a description and icon:
+1. Go to the **Display details** tab.
+2. In the **Short description** section, add a one-sentence description of your app to make your users aware of what your app does, and save your changes. You can change the description later.
+3. Add an icon for your app and save your changes. Your icon will be displayed in the Marketplace and the Agent App. You can drag and drop your image or browse your file manager for it.
+
+<img
+  alt="Creating LiveChat apps step3"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step3.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 4 Listing details**
+
+The next step is adding a few details of your app’s functionalities to make it easier to understand for your users. 
+
+To do that, go to the **Listing details** tab and fill in all the details.
+
+<SectionLink to={"/monetization/app-review-process/#app-details"}>
+  How to fill in app details
+</SectionLink>
+
+<img
+  alt="Creating LiveChat apps step4"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step4.png"
+  width="1515px"
+  height="819px"
+/>
+
+### **Step 5 Test your application**
+
+Before you publish your application on the Marketplace, you need to test it first to ensure that no bugs or errors appear. 
+
+To do that, you need to install your app privately on your LiveChat license by going to the **Private installation** tab and clicking the **Install app** button.
+
+<img
+  alt="Creating LiveChat apps step5"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step5.png"
+  width="1515px"
+  height="819px"
+/>
+
+When the installation is done, you can return to your LiveChat Agent App and refresh the website. After doing that, you’ll see your widget. 
+
+Please remember that your widget placement in the LiveChat Agent App will depend on which widget placement you've chosen while creating the new Agent App Widget.
+
+<SectionLink to={"/monetization/app-review-process/#common-problems"}>
+  Take a look at things to remember when testing the app.
+</SectionLink>
+
+<img
+  alt="Creating LiveChat apps step5.1"
+  src="/images/getting-started/livechat-apps/LiveChat-apps step5.1.png"
+  width="1499px"
+  height="596px"
+/>
+
+Congratulations!
+Now, you can enjoy your simple application under your license. If you want to build a public app, please take a look at the next step, where you’ll find out how to submit your application for review.
+
+### **Step 6 Send your application for review (optional)**
```

Comment:
@oliwiapolec, do you think we should keep this section? Maybe it would be better to leave one sentence and redirect to the App review process document?

Reply:
App review process is quite complex so one paragraph doesn't suffice to explain it, and at the same time, we want to keep this tutorial simple so adding extra info about app review would be counter-productive